src/partials/marketStats/marketStats.vue

```diff
@@ -39,7 +39,10 @@ export default {
     'priceChf',
     'marketCapChf',
     'priceBtc'
-  ])
+  ]),
+  mounted () {
+    pollAction('_marketStats/get')
```

Comment:
This call doesn't have sense because `pollAction` function just returns an object with `mounted`, `beforeDestroy` methods without doing anything else.